### PR TITLE
Log improvements + code cleanup + fixes

### PR DESCRIPTION
**Pull request by addb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/main/media/mediaSession.brs [hunk 1]
```diff
@@ -198,10 +201,17 @@ function _adb_MediaSession(clientSessionId as string, configurationModule as obj
                                 if not _adb_isEmptyOrInvalidArray(responseObj.errors) then
                                     m._processEdgeResponseErrors(responseObj.errors)
                                 end if
+                            else
+                                ''' Should execute this code when there is a non-recoverable error for sessionStart request
+                                ''' Abort the session
+                                m.close(true)
+                                _adb_logVerbose("_processEdgeRequestQueue() - SessionStart request failed with unrecoverable error.")
```
Comment (praveek) [inline code/main/media/mediaSession.brs:208]: Can this be a warning?

File: code/main/media/mediaSession.brs [hunk 2]
```diff
@@ -210,8 +220,18 @@ function _adb_MediaSession(clientSessionId as string, configurationModule as obj
         _processEdgeResponseHandles: function(handleList as object) as void
             for each handle in handleList
                 if handle.type = m._HANDLE_TYPE_SESSION_START
-                    m._backendSessionId = handle.payload[0]["sessionId"]
+                    payloadSessionId = handle.payload[0]["sessionId"]
+
+                    if _adb_isEmptyOrInvalidString(payloadSessionId)
+                        m.close(true)
+                        _adb_logVerbose("_processEdgeResponseHandles() - SessionStart request returned with empty or invalid sessionID.")
```
Comment (praveek) [inline code/main/media/mediaSession.brs:227]: Can you also make this a warning?

File: code/main/media/mediaSession.brs [hunk 3]
```diff
@@ -224,6 +244,7 @@ function _adb_MediaSession(clientSessionId as string, configurationModule as obj
                 if error.type = m._ERROR_TYPE_VA_EDGE_400
                     ''' abort the session if sessionStart fails
                     m.close(true)
+                    _adb_logVerbose("_processEdgeResponseErrors() - Closing the session as the SessionStart request failed.")
```
Comment (praveek) [inline code/main/media/mediaSession.brs:247]: Can you make this an error?

File: code/main/services/networkService.brs [hunk 4]
```diff
@@ -86,11 +86,11 @@ function _adb_NetworkService() as object
                         responseCode = msg.GetResponseCode()
                         responseString = msg.getString()
                         failureMessage = msg.GetFailureReason()
-                        _adb_logDebug("syncPostRequest() -  Received response code:(" + FormatJson(responseCode) + ") body:(" + FormatJson(responseString) + ") message:(" + FormatJson(failureMessage) + ").")
+                        _adb_logDebug("syncPostRequest() -  Received response code:(" + FormatJson(responseCode) + ") body:("  + chr(10) + responseString  + chr(10) + ") message:("  + chr(10) + failureMessage  + chr(10) + ").")
```
Comment (praveek) [inline code/main/services/networkService.brs:89]: Why not use `\n` instead of chr(10)?

Comment (addb) [reply code/main/services/networkService.brs:89]: Let me try that. I thought that doesn't work.

Comment (addb) [reply code/main/services/networkService.brs:89]: \n doesn't work.

**Review by praveek — APPROVED** [review]

**Pull request merged**